source/arroost/components/tunnel.js

```diff
@@ -36,6 +36,11 @@ export class Tunnel extends Component {
 	 */
 	static tunnels = new Map()
 
+	/**
+	 * @type {Set<Tunnel>}
+	 */
+	static inViewTunnels = new Set()
```

Comment:
we keep track of all on screen tunnels, so we can iterate through them when zooming in/out